tox.ini

```diff
@@ -4,22 +4,37 @@
 [tox]
 isolated_build = True
 skip_missing_interpreters = True
-envlist = build-json-schemas, unit, lint
+envlist = build-json-schemas, unit, lint, charm-pov
 
 
 [vars]
 tst_path = {toxinidir}/tests
 docs_path = {toxinidir}/docs
+tester_path = {toxinidir}/tester
 all_path = {[vars]tst_path} {[vars]docs_path}
 
 [testenv:unit]
 description = run unittests
 deps =
-    pytest
+    .[unit_tests]
+    git+https://github.com/PietroPasotti/interface-tester-pytest
 setenv =
     PYTHONPATH={toxinidir}
 commands =
-    pytest {[vars]tst_path}
+    pytest {[vars]tst_path}/test_unit
+
+
+[testenv:charm-pov]
```

Comment:
Maybe document this behavior somewhere, it's not clear who should use it, when

Reply:
what behaviour? Do you mean the charm-pov test environment?

Reply:
Yes sorry my comment wasn't clear. It's not clear what's this `charm-pov`, what it is, when should it be used, etc.

Reply:
done